apps/dashboard/app/api/auth/[...nextauth]/route.ts

```diff
@@ -52,6 +53,6 @@ export const authOptions = {
   },
 };
 
-const handler = NextAuth(authOptions)
+const handler = NextAuth(authOptions);
```

Comment:
why are we changing to use `;`? we need to be consistent across apps

Reply:
it's because of the default nextjs formatting, but I agree we need to have consistency in our linting. it was not apparent before because nextjs was used on different repos